extensions/binding/org.eclipse.smarthome.binding.bosesoundtouch/src/main/java/org/eclipse/smarthome/binding/bosesoundtouch/internal/ContentItem.java

```diff
@@ -71,7 +71,7 @@ public boolean isValid() {
         if (getOperationMode() == OperationModeType.STANDBY) {
             return true;
         }
-        if (itemName.equals("") || source.equals("")) {
+        if (itemName == null || source == null || itemName.equals("") || source.equals("")) {
```

Comment:
While you are at it, maybe you could also change the `.equals("")` to `.isEmpty()`?

Reply:
Good point. Done!